Resources/public/js/views/fields/ez-maplocation-editview.js

```diff
@@ -83,6 +87,25 @@ YUI.add('ez-maplocation-editview', function (Y) {
         },
 
         /**
+         * Set the description tooltip position in Y axis.
+         * The tooltip position is modified by it's height.
+         *
+         * @method _setTooltipPosition
+         * @protected
+         */
+        _setTooltipPosition: function () {
```

Comment:
why is it overridden ? as far as I can see, you just have to add the `ez-editfield-input` class on the address input to be able to use the default implementation